lib/glific/providers/gupshup/template.ex

```diff
@@ -249,7 +249,7 @@ defmodule Glific.Providers.Gupshup.Template do
   defp is_valid_language?(_language), do: {:error, "Invalid Language"}
 
   @spec is_valid_category?(String.t()) :: true | {:error, String.t()}
-  defp is_valid_category?(category) when category in ["TRANSACTIONAL", "MARKETING", "OTP"],
+  defp is_valid_category?(category) when category in ["UTILITY", "MARKETING", "AUTHENTICATION"],
```

Comment:
This does not make sense to duplicate the categories. Can you use the same function [session_templates.ex](https://github.com/glific/glific/pull/2724/files#diff-5441acb5a895432a63f13fca78fb8c95f1ff9f24faeb5791f900c0fdcf3e0d63).list_whatsapp_hsm_categories ?